### DISCUSSION TITLE
WRF Hydro Standalone vs Coupled

### DISCUSSION THREAD

**Discussion by zhong-industries** [NCAR/wrf_hydro_nwm_public#761] [General]
I was able to get WRF Hydro running in standalone mode, but I was wondering if I should also get it to work coupled to the WRF model. What are the benefits (if any) of running WRF Hydro as a coupled model? And vice versa, are there any benefits to running standalone?

I did find slides about this here: https://ral.ucar.edu/sites/default/files/public/projects/wrf-hydro/training-materials/ams2021shortcourse/5-wrf-hydro-implementation-bestpractices-jan-2021-abridged.pdf and slide 2 specifically talks about coupled and uncoupled, but I would like to know the best practice of which to use and how y'all intended each version to be used.

**Comment by scrasmussen** [comment]
Hi, choosing between the stand-alone uncoupled mode vs. coupled with WRF depends on a number of factors, some of which are:

- Area of research: If the primary interest is in detailed hydrological processes without the need for detailed atmospheric feedbacks, the uncoupled mode may be more appropriate. 
- Computational Cost: The coupled mode requires more computational power and resources
- Data Availability: Uncoupled WRF-Hydro runs the hydrologically enhanced LSM using idealized data or a user’s specified atmospheric forcing

[This resource](https://ral.ucar.edu/sites/default/files/docs/water/howtobuildrunwrfhydrov511instandalonemode.pdf) has good information on the uncoupled process.

**Reply by zhong-industries** [reply]
Thanks for the response Soren. Those all seem to be advantages for the uncoupled mode however. Are there any cases to use the coupled mode instead?

**Reply by aubreyd** [reply]
Hi Alfred: You would want to use the fully 2-way WRF/WRF-Hydro coupled mode if you are interested in feedbacks between the land surface hydrology (e.g., better representation of soil moisture states and ET due to lateral redistribution) and the atmosphere. There are a number of studies in the literature that provide great examples.
